tests/filesystem/FileTest.yml

```diff
@@ -42,9 +44,12 @@ Group:
   cmsusers:
     Title: 'CMS Users'
     Permissions: =>Permission.cmsmain
+  securityusers:
+    Title: 'Security Users'
+    Permissions: =>Permission.securityadmin
   assetusers:
     Title: 'Asset Users'
-    Permissions: =>Permission.cmsmain, =>Permission.assetadmin
```

Comment:
This was a bug; Relation between groups and permissions is has_many not many_many, and two groups were attempting to link to the same permission.